src/builder/index.ts

```diff
@@ -23,6 +23,7 @@ export async function buildSW(
     documentCachingOptions: {},
   },
   importFrom: string = 'https://cdn.ampproject.org/amp-sw.js',
+  eject: boolean = false,
```

Comment:
Let's use the same name everywhere, called `eject` here and `installNoOpServiceWorker` below.

`eject` seems like a cleaner word given what it does.